README.md

```diff
@@ -53,26 +53,34 @@ Reference: [docs](https://probot.github.io/docs/configuration/)
 
 ### Methods of developments
 
-1. In Local system
+1. In local system
+   1. It's crucial your node version matches the one installed in the container deployment.
+      Using different version can make any packages installed in your development unusable when built into a container image.
+      Please install [NVM](https://github.com/nvm-sh/nvm) and set it up for NodeJS v16:
 
-    Start the server for services.
+      ```sh
+      # Install currently locked version from .nvmrc
+      nvm install
+      # Use this version
+      nvm use
+      ```
 
-    ```sh
-        # Install dependencies
-        npm install
+      Note: `nvm use` can be called automatically by your shell on entering the directory, see docs on [NVM](https://github.com/nvm-sh/nvm#deeper-shell-integration) or use a plugin like [oh-my-zsh nvm](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/nvm)
 
-        # Build all the requirements
-        npm run build
+   2. Remember to set environment variables via `.env` file. We require:
```

Comment:
Don't we also need to set the `PRIVATE_KEY` variable?

Reply:
Not necessarily see:
https://probot.github.io/docs/development/#manually-configuring-a-github-app

![image](https://user-images.githubusercontent.com/7453394/164290789-cea2c6d8-6f6b-4422-9c97-208308b7a026.png)

However we should specify that the priv key has to be located in the folder... I'll update the PR.